contribution.md

```diff
@@ -0,0 +1,38 @@
+# How to contribute.
```

Comment:
I have checked a few different project and it seems like `CONTRIBUTING.md` is the usual name - can you please rename this file?

Reply:
renamed